### PR TITLE
cam6_3_153: Update namelist settings

### LINKED ISSUE #954: Remove README_EXTERNALS
### Issue Type

Documentation Update

### Issue Description

Now that we are discussing a possible replacement for manage_externals, perhaps it is time to finally remove the svn-based README_EXTERNALS file?

### Will this change answers?

No

### Will you be implementing this yourself?

No

### LINKED ISSUE #976: Bring in namelist settings which Cecile is using for current testing
### Issue Type

Other (please describe below)

### Issue Description

The list of namelist settings which @cecilehannay is currently using and need to be changed are:

! dust tuning
dust_emis_fact = 1.3
! clubb diffusion on
clubb_l_do_expldiff_rtm_thlm = .true.
! tuning
microp_aero_wsub_scale  = 1.0
clubb_C8=4.25
! revert NorESMs mods
cldfrc2m_do_avg_aist_algs = .false.
cldfrc2m_qist_min = 1.e-7
bnd_topo = "/glade/campaign/cgd/amp/pel/topo/files/se/ne30pg3_gmted2010_modis_bedmachine_nc3000_Laplace0100_noleak_20240117.nc"
se_hypervis_subcycle = 2

@adamrher, @PeterHjortLauritzen - Should these changes all be tied to` -phys cam_dev` or made default for all CAM runs?

### Will this change answers?

Yes

### Will you be implementing this yourself?

Yes

### LINKED ISSUE #988: CAM no longer builds with intel-oneapi compilers
### What happened?

When attempting to build a case created with CAM for the atmosphere and `intel-oneapi` for the compiler, builds fail. 

Example error message:
> `ERROR: Command: '/glade/work/gdicker/EarthWorks/EWRepo_Testing/2024Mar08_CAMIssue35/CAM/CAM/bld/configure -s -fc_type intel-oneapi -dyn eul -hgrid 64x128 -cpl nuopc -usr_src /glade/work/gdicker/EarthWorks/EWRepo_Testing/2024Mar08_CAMIssue35/CAM/CAM/FHS94.check.intel-oneapi/SourceMods/src.cam -nlev 30 -spmd -nosmp -ocn socn -phys held_suarez -analytic_ic' failed with error 'ERROR: intel-oneapi is not a valid value for parameter fc_type: valid values are cray,pgi,intel,gnu,pathscale,ibm,nag,nvhpc,pgi-gpu,nvhpc-gpu' from dir '/glade/work/gdicker/EarthWorks/EWRepo_Testing/2024Mar08_CAMIssue35/CAM/CAM/FHS94.check.intel-oneapi/Buildconf/camconf'`

### What are the steps to reproduce the bug?

1. Get a clone of CAM that is based on cam6_3_147 or newer, and run `checkout_externals`
2. Create and setup a case that uses CAM and requests `intel-oneapi` as a compiler
    - e.g. `./cime/scripts/create_newcase --res T42z30_T42_mg17 --case FHS94.check.intel-oneapi --project ${YOUR_PROJECT} --compset FHS94 --compiler intel-oneapi`
3. Attempt to run `./case.build` from within the case and see an error like above

### What CAM tag were you using?

cam6_3_147 or newer

### What machine were you running CAM on?

CISL machine (e.g. cheyenne)

### What compiler were you using?

Intel

### Path to a case directory, if applicable

_No response_

### Will you be addressing this bug yourself?

Any CAM SE can do this

### Extra info

_No response_

### PR DESCRIPTION
**Pull request by cacraigucar**
Closes #976 

Closes #954 

Closes #990 which closes #988

This PR does not bring in the setting for se_hypervis_subcycle or the bnd_topo file as those will be brought in via #968

### CODE REVIEW

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 1]
```diff
@@ -2177,6 +2178,7 @@
 <clubb_l_damp_wp2_using_em                        > .false. </clubb_l_damp_wp2_using_em>
 <clubb_l_damp_wp3_Skw_squared                     > .false. </clubb_l_damp_wp3_Skw_squared>
 <clubb_l_do_expldiff_rtm_thlm                     > .false. </clubb_l_do_expldiff_rtm_thlm>
+<clubb_l_do_expldiff_rtm_thlm phys="cam_dev"      > .true.  </clubb_l_do_expldiff_rtm_thlm>
```
Comment (adamrher) [inline bld/namelist_files/namelist_defaults_cam.xml:2181]: Just noting here that turning this on changes the climate to be more like cam tag 058, the last time it was on. It's a big change.

**Review by adamrher — APPROVED** [review]
Looks good to me. 

Are gust mod namelists controlled by share or some other external?

**Comment by cacraigucar** [comment]
> Looks good to me.
> 
> Are gust mod namelists controlled by share or some other external?

@peverwhee just made cam6_3_150 with the convective gustiness PR #943 - Is this what you are asking about?

**Comment by adamrher** [comment]
Yes I was asking about PR https://github.com/ESCOMP/CAM/pull/943, thanks for providing that. I see the CMEPS external is turning on the gust param. when running the ```CAM%DEV``` compset. That's all I needed thanks!

**Review by nusbaume — APPROVED** [review]

**Comment by cacraigucar** [comment]
@adamrher, @PeterHjortLauritzen and @cecilehannay approved the namelist changes

**Pull request merged**